src/Steam.Common/GameId.cs

```diff
@@ -0,0 +1,188 @@
+using System;
+
+namespace Steam
+{
+    /// <summary>
+    /// Definition of a GameId, interpreted from a <see cref="ulong"/> value
+    /// </summary>
+    public readonly struct GameId : IEquatable<GameId>
+    {
+        /// <summary>
+        /// Generic shortcut gameid, contains an empty AppId
+        /// </summary>
+        public static readonly GameId Shortcut = new GameId(new App(), GameTypes.Shortcut);
+
+        private readonly ulong _value;
+
+        /// <summary>
+        /// the app id of the game
```

Comment:
Consistent capitalization in docs would be nice. This applies to other docs as well.